source/schedulexmlnode.d

```diff
@@ -0,0 +1,249 @@
+module scheduleXmlNode;
+
+import std.datetime;
+import kxml.xml : XmlNode;
+import std.array : empty, front;
+
+import xmlconstants;
+import xmlparserexceptions;
+
+public: 
+struct ScheduleXmlNode {
+    // The innerNode should not be writeable, but alias this needs access.
+    private XmlNode _innerNode;
+    public XmlNode innerNode() @property {
```

Comment:
don't annotate every member with `public`, `private` etc., use the `public:` syntax instead. (this is not Java :))